build/Tests.lua

```diff
@@ -18,11 +18,11 @@ function SetupExampleProject()
   SetupParser()
 end
 
-function SetupTestProject(name, extraFiles)
+function SetupTestProject(name, extraFiles, extension)
```

Comment:
Rename "extension" to "suffix" because of the precise meaning "extension" has for files. Rename it everywhere.